app/assets/javascripts/qsets.js

```diff
@@ -94,3 +94,11 @@ function initQuestionFilter() {
     $('.no-questions > p').html(noQuestionsNotification);
   }
 }
+
+function showNoSubsetNotification() {
+  if ($('.qset').length == 0) {
+    $('.no-qsets').removeClass("hidden");
+    var noSubsetsNotification = 'There are no subsets. ' + '<a href="#" data-toggle="modal" data-target="#modal-new-qset">Create one now!</a>';
```

Comment:
Good point, yeah I just implemented it the same way the "Create one now" link is for pages with no questions. Elliott, what do you think?